service/src/main/kotlin/fi/espoo/evaka/messaging/message/MessageAccountQueries.kt

```diff
@@ -67,6 +68,7 @@ FROM message_account acc
     LEFT JOIN daycare_group dg ON acc.daycare_group_id = dg.id
     LEFT JOIN daycare dc ON dc.id = dg.daycare_id
     LEFT JOIN daycare_acl acl ON acl.daycare_id = dg.daycare_id
+    LEFT JOIN daycare_group_acl gacl ON gacl.daycare_group_id = dg.id
```

Comment:
Redundant join? Actually, it seems like both of these acl joins are redundant now.

Reply:
Good catch, fixed!